src/config.example.json

```diff
@@ -158,7 +158,9 @@
     "icons": {
         "Default": {
             "path": "/img/",
-            "gymAnchorY": 0.917,
+            "raidOffsetY": 0.269,
+            "questOffsetY": 0,
```

Comment:
Again, just to emphasize that these two new offsets are optional. A default value was supplied in `index.js`. However, the update to `gymAnchorY` is probably necessary to make gym anchor correctly.